docs/deployment/primary-branch.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: primary-branch
+title: Set a primary branch
+hide_title: true
+description: Set your primary or default branch to ensure Semgrep full scans display accurate counts and deduplicated findings.
+tags:
+  - Core deployment
+---
+
+# Set a primary branch
+
+A **primary branch** is the base or target branch for pull and merge requests. It is usually referred to as a **default branch** or **trunk** by your source code manager (SCM). Typical names for a primary branch include `dev`, `production`, or `develop`.
+
+In many cases, Semgrep automatically detects primary branches when they first scan your project. If you have projects (repositories) with unique primary branch names, you can set them through the Semgrep web app.
```

Comment:
```suggestion
In many cases, Semgrep automatically detects primary branches when it first scans your project. If you have projects (repositories) with unique primary branch names, you can set them through the Semgrep web app.
```

Reply:
@khorne3 This was meant to be Semgrep doing the scan